packages/compass-sidebar/src/components/multiple-connections/connections-navigation.tsx

```diff
@@ -291,6 +293,7 @@ const ConnectionsNavigation: React.FC<ConnectionsNavigationProps> = ({
           return;
         case 'open-shell':
           openShellWorkspace(item.connectionInfo.id, { newTab: true });
+          track('Shell Open', { entrypoint: 'sidebar' }, item.connectionInfo);
```

Comment:
Maybe we don't use a `'Shell '`-prefixed name at all if we want to avoid collisions entirely?

Reply:
You're right, originally I thought I am following a convention but it's safer to reserve that convention for the in-shell events